TS_src/server/server.ts

```diff
@@ -17,7 +17,7 @@ const registerRouter = require('./routers/registerRouter');
 const contactsRouter = require('./routers/contactsRouter');
 const usersRouter = require('./routers/usersRouter');
 const tripsRouter = require('./routers/tripsRouter');
-import authRouter from './routers/authRouter'
+// import authRouter from './routers/authRouter'
```

Comment:
In the dev branch, the authRouter is being used for google oauth login. Does the contacts feature work when the authRouter is implemented?